redpanda/manifest.json

```diff
@@ -20,9 +20,9 @@
     ]
   },
   "author": {
-    "support_email": "support@vectorized.io",
+    "support_email": "support@redpanda.com",
```

Comment:
Do you need to also update the contact info in [CODEOWNERS](https://github.com/DataDog/integrations-extras/blob/d36f3b3683a68010ef33faa777cc6f3d896e207e/.github/CODEOWNERS#L131)?

Reply:
Thanks @yzhan289 , updated that. Checking it needs to be a person rather than an organisation? I'm now seeing erros.

Reply:
On https://github.com/DataDog/integrations-extras/pull/1864/files it showed CODEOWNERS errors, but the CI tests pass.

Reply:
I believe those errors are fine to ignore.